testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/ModuleTestCase.java

```diff
@@ -186,6 +186,20 @@ public void addRemoveModuleMetaInf() throws Exception {
         testModuleAndSlotExist(false, slot, getModulePath());
     }
 
+    @Test
+    public void addModuleWithWhitespacesInDependencies() throws Exception {
+        // should trim the surrounding whitespaces
+        try {
+            cli.sendLine("module add --name=" + MODULE_NAME
+                    + " --resources=" + jarFile.getAbsolutePath()
+                    + " --dependencies=[ org.jboss.logging ]"
```

Comment:
The current module.txt says:

```
NOTE: In non-interactive mode this list must be surrounded by square brackets, e.g. [dep1,dep2].
```

If that's no longer true, great! But we need to be sure that the [] syntax still works since it was documented.